src/scripts/main.js

```diff
@@ -1,3 +1,29 @@
 'use strict';
 
 // write your code here
+const elements = document.getElementsByClassName('population');
+
+const populations = Array.from(elements).map((element) => {
+  return parseInt(element.innerText.replace(/,/g, ''), 10);
+});
+
+const averagePopulation = document.querySelector('.average-population');
+
+averagePopulation.innerText = averagePeople(populations);
```

Comment:
The function `averagePeople` is correctly used here, but ensure that the function is defined and implemented correctly. It seems to be correct based on the provided code.